common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -1279,9 +1280,9 @@ object PageElement {
           embedType <- wtd.witnessEmbedType
         } yield {
           embedType match {
-            case "image" => Some(makeWitnessBlockElementImage(element, wtd))
-            case "video" => Some(makeWitnessBlockElementVideo(element, wtd))
-            case "text"  => Some(makeWitnessBlockElementText(element, wtd))
+            case "image" => Some(makeWitnessBlockElementImage(element, wtd, isLiveblog))
```

Comment:
Its kind of interesting that we do add the tracking field to the witnessBlock, we don't add a click to view wrapper in DCR even if it were true.